static/app/views/performance/vitalDetail/vitalsComparison.tsx

```diff
@@ -0,0 +1,227 @@
+    default:
+      return 'success';
+  }
+}
+
+function MetricsCard({
+  title,
+  vital,
+  value,
+  tooltip,
+}: {
+  title: string;
+  tooltip: string;
+  value: number;
+  vital: WebVital;
+}) {
+  // round to 2 decimals if <10s, otherwise use just 1 decimal
+  const score = getScore(vital, value);
+  const numDecimals = value >= 10_000 ? 1 : 2;
+  const timeInSeconds = value / 1000.0;
+  return (
+    <MetricsCardWrapper>
+      <MetricsTitle>
+        {title} (p75) <StyledQuestionTooltip title={tooltip} size="xs" />
+      </MetricsTitle>
+      <ScoreWrapper>
+        <ScoreContent>{timeInSeconds.toFixed(numDecimals)}s</ScoreContent>
+        <TagWrapper>
+          <StyledTag type={getTagLevel(score)}>{getIndicatorString(score)}</StyledTag>
+        </TagWrapper>
+      </ScoreWrapper>
+    </MetricsCardWrapper>
+  );
+}
+
+function ContentWrapper({
+  organization,
+  vital,
+  children,
+  count,
+}: {
+  children: React.ReactNode;
+  count: number;
+  organization: Organization;
+  vital: WebVital;
+}) {
+  useEffect(() => {
+    trackAdvancedAnalyticsEvent('performance_views.vital_detail.comparsion_viewed', {
+      organization,
+      vital,
+      count,
+    });
+  });
+  return <Container>{children}</Container>;
+}
+
+function VitalsComparison(props: Props) {
+  const {location, vital: _vital, organization} = props;
+  const vitals = Array.isArray(_vital) ? _vital : [_vital];
+  const vital = vitals[0];
+  if (!SUPPORTED_VITALS.includes(vital)) {
+    return null;
+  }
+  return (
+    <VitalsCardDiscoverQuery location={location} vitals={vitals}>
+      {({isLoading, vitalsData}) => {
+        if (isLoading || !vitalsData) {
+          return null;
+        }
+        const {p75} = vitalsData[vital];
+        if (!p75) {
+          return null;
+        }
+        const lookupName = vital === 'measurements.fcp' ? 'FCP' : 'LCP';
+        const sentryStandard = SENTRY_CUSTOMERS[lookupName];
+        const industryStandard = INDUSTRY_STANDARDS[lookupName];
+        const count = vitalsData[vital].total;
+        // only show it if we hit the min number
+        if (count < MIN_VITAL_COUNT_FOR_DISPLAY) {
+          return null;
+        }
+        return (
+          <ContentWrapper {...{organization, vital, count}}>
+            <MetricsCard
+              title={t('Selected Projects')}
+              vital={vital}
+              value={p75}
+              tooltip={tct(
+                "25% of your project's transactions have an [lookupName] greater than this number. Good, Bad, Meh segmentation is based on Google industry standards.",
+                {lookupName}
+              )}
+            />
+            <MetricsCard
+              title={t('Sentry Peers')}
+              vital={vital}
+              value={sentryStandard}
+              tooltip={tct(
+                'Calculated as 50th percentile of top 33% p75 [lookupName] time across all web projects instrumenting their performance with Sentry.',
```

Comment:
I think we may need to revise this tooltip, the way the calculation is worded seems quite confusing: `50th percentile of top 33% p75 [lookupName] time`

Reply:
@0Calories Yea, let me check with folks on the copy

Reply:
How about the below?

Of the 50th percentile of web projects sending transactions, this is the top 33% p75 [lookupName] time.

Reply:
@pevensentry So is this the top 16.5% band? Or is this 50-66.5%? Now I'm confused lol.

Reply:
@0Calories Went with `'20% of Sentry customers have a p75 [lookupName] lower than this.`

Rounding to 20% because rounded numbers are nice

Reply:
Nice! That sounds much clearer